payload/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Script assumes python venv is already properly configured
+# required env vars:
+# $BOT_PAT
+# $WHEEL_RELEASE
+# $WHEEL_BASEURL
+set -ex
```

Comment:
Might be worth setting `-u` as well to make sure all referenced variables are set